rust/tvm-rt/src/to_function.rs

```diff
@@ -103,8 +103,10 @@ pub trait ToFunction<I, O>: Sized {
                 value = args_list[i];
                 tcode = type_codes_list[i];
                 if tcode == ffi::TVMArgTypeCode_kTVMObjectHandle as c_int
+                    || tcode == ffi::TVMArgTypeCode_kkTVMObjectRValueRefArg as c_int
```

Comment:
looks like there is an extra k 
```suggestion
                    || tcode == ffi::TVMArgTypeCode_kTVMObjectRValueRefArg as c_int
```